packages/react/src/combobox/utils.ts

```diff
@@ -90,3 +90,16 @@ export function generateHighlightStyles(
 
 	return styles;
 }
+
+export function useIsIos() {
+	const [isIos, setIsIos] = useState(false);
+
+	useEffect(() => {
+		// See https://github.com/stowball/Layout-Engine/blob/master/layout.engine.js#L86
+		if (CSS.supports('-webkit-overflow-scrolling', 'auto')) {
```

Comment:
I just took a look on [caniuse for -webkit-overflow-scrolling auto](https://caniuse.com/?search=-webkit-overflow-scrolling%20auto) which is suggesting it's now supported in Android as of this year. Have you checked this?

Reply:
I have v128 and it's not supported, by I have just updated it to also test for the apple-pay-button. It shouldn't make any difference if Android does return true, because there's no way AFAIK for you to blur without a next target like on iOS